internal/shell/shell_cmd_test.go

```diff
@@ -152,3 +152,14 @@ func assertEqualScope(t *testing.T, expected, actual rel.Scope) {
 		assert.True(t, v1.(rel.Value).Equal(v2))
 	}
 }
+
+// `//arrai.info` can't be constructed in this test as it involves package `main`.
+func removedVoidItems(scope rel.Scope) rel.Scope {
```

Comment:
Nit: This name is slightly misleading: it's removing a very specific item (arrai.info), not all void items as the name suggests. Maybe `removeArraiInfo()`?

Also nit: s/removed/remove/
```suggestion
// removeArraiInfo removes `//arrai.info` from the scope as it can't be constructed in this test due to its lack of package `main`.
func removeArraiInfo(scope rel.Scope) rel.Scope {
```